source/change-streams/change-streams.rst

```diff
@@ -1100,3 +1093,5 @@ Changelog
 +------------+------------------------------------------------------------+
 | 2022-08-22 | Added ``clusterTime`` to ``ChangeStreamDocument``.         |
 +------------+------------------------------------------------------------+
+| 2022-10-05 | Remove spec front matter.                                  |
```

Comment:
I'll note that the format of this changelog is different from all others.

Reply:
Indeed. I called this out in the commit message along with the CSFLE changelog syntax.

There was enough effort put into both files that I reasoned it wasn't worth changing them in this PR.